src/bootstrap/build/sanity.rs

```diff
@@ -78,6 +79,24 @@ pub fn check(build: &mut Build) {
             panic!("the iOS target is only supported on OSX");
         }
 
+        if cfg!(windows) {
```

Comment:
It is not need for `llvm-config` on Windows, since it's in the directory with the DLLs. It is needed for executables which are not in the LLVM's binary directory which link dynamically to LLVM (like rustc).